### PR TITLE
feat: Add the Vulture router.

### PR DESCRIPTION
**Pull request by loganrockmore**

### CODE REVIEW

File: lib/router.js [hunk 1]
```diff
@@ -2037,4 +2037,16 @@ router.get('/mastodon/timeline/:site/:only_media?', require('./routes/mastodon/t
...
+router.get('/vulture/tv', require('./routes/vulture/tv'));
+router.get('/vulture/movies', require('./routes/vulture/movies'));
+router.get('/vulture/comedy', require('./routes/vulture/comedy'));
+router.get('/vulture/music', require('./routes/vulture/music'));
+router.get('/vulture/tvrecaps', require('./routes/vulture/tvrecaps'));
+router.get('/vulture/books', require('./routes/vulture/books'));
+router.get('/vulture/theater', require('./routes/vulture/theater'));
+router.get('/vulture/art', require('./routes/vulture/art'));
+router.get('/vulture/awards', require('./routes/vulture/awards'));
+router.get('/vulture/video', require('./routes/vulture/video'));
```
Comment (DIYgod) [inline lib/router.js]:
```js
router.get('/vulture/:type', require('./routes/vulture/index'));
```

File: lib/routes/vulture/art.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const utils = require('./utils');
+
+module.exports = async (ctx) => {
+    const url = 'https://www.vulture.com/art/';
```
Comment (DIYgod) [inline lib/routes/vulture/art.js]:
```js
const url = `https://www.vulture.com/${ctx.params.type}/`;
```

File: lib/router.js (hunk 1 above)
Comment (loganrockmore) [reply lib/router.js]: Makes total sense, thanks! Fixed!

**Comment by loganrockmore** [comment]
@DIYgod it looks like this failed Travis CI, but for reasons unrelated to the code change.  When I run `npm run test` locally on master, the same test fails.  Any suggestions on what to do here?  Thanks!

**Comment by DIYgod** [comment]
> @DIYgod it looks like this failed Travis CI, but for reasons unrelated to the code change. When I run `npm run test` locally on master, the same test fails. Any suggestions on what to do here? Thanks!

It's not Travis CI failure, it is a git conflict, don't mind.

**Comment by loganrockmore** [comment]
Okay, thank you so much!

**Pull request merged**